src/structures/GuildAuditLogs.js

```diff
@@ -349,7 +351,7 @@ class GuildAuditLogsEntry {
 
     /**
      * Any extra data from the entry
-     * @type {?Object|Role|GuildMember}
+     * @type {?(Object|Role|GuildMember|Channel)}
```

Comment:
This doesn't look like it could be a Channel to me:
```suggestion
     * @type {?(Object|Role|GuildMember)}
```

Reply:
hmm, right. It can only be a property of the generic object at the most. Thanks 👍